spec/player_spec.rb

```diff
@@ -6,11 +6,11 @@
     @player2 = Scrabble::Player.new("andre")
     @player3 = Scrabble::Player.new("batman")
     @player4 = Scrabble::Player.new("spiderman")
+    @player5 = Scrabble::Player.new("superman")
```

Comment:
I'm not sure I understand why there are five separate `Player` instances in this spec file. Because each `it` block test is run completely independent of all other tests, none of the code from the other ones will impact the `@player1` instance.

In general you only need to write a spec with more than one instance of a class if at least one test requires that multiple instances of the class interact with each other. Otherwise, it's perfectly fine to use the same instance from a `before` block in every test.